pkg/collector/corechecks/ebpf/probe/ebpfcheck/probe.go

```diff
@@ -523,3 +525,80 @@ func ringBufferMemoryUsage(mapStats *model.EBPFMapStats, info *ebpf.MapInfo, k *
 	mapStats.RSS = mapStats.MaxSize
 	return nil
 }
+
+func hashMapNumberOfEntries(mp *ebpf.Map) int64 {
+	if isPerCPU(mp.Type()) {
+		return -1
```

Comment:
@brycekahle how would `-1` get interpreted in the backend?

Reply:
The collector discards those results, I think they shouldn't get to the backend https://github.com/DataDog/datadog-agent/blob/bd281b4b0d41de85c287416cf3b4c7f6b91afdae/pkg/collector/corechecks/ebpf/ebpf.go#L122